### DISCUSSION TITLE
Keychain - Bookmark management (AEIP-12 )

### DISCUSSION THREAD

**Discussion by samuelmanzanera** [archethic-foundation/aeip#39] [AEIP]
This is a discussion for the AEIP12 located at: https://github.com/archethic-foundation/aeip/blob/main/AEIP-12.md

**Comment by bchamagne** [comment]
> { archethic:keychainAddress": 

I think, in the example, we should replace `keychainAddress` by an example address.

> Note that bookmark information is optional. In that case, no information is present in the secret. It isn't necessary to provide an empty bookmark.

I don't understand what is this information.

**Reply by redDwarf03** [reply]
Ok for example. I updated it. Thx
About the comment on optional info, I just would like to explain this AEIP is not mandatory in a keychain transaction. So if user doesn’t want bookmark in his keychain, we will have no secret instead a secret with an empty bookmark json
